From 8c229270f6bae83b705a03714c46067a7aa57b02 Mon Sep 17 00:00:00 2001
From: Michael Opdenacker <michael.opdenacker@bootlin.com>
Date: Tue, 19 Sep 2023 13:14:29 +0200
Subject: [PATCH] topology: include locale.h

This solves an issue compiling with the musl libc.

Fixes: https://github.com/alsa-project/alsa-utils/issues/239
Signed-off-by: Michael Opdenacker <michael.opdenacker@bootlin.com>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>

Upstream: https://github.com/alsa-project/alsa-utils/commit/8c229270f6bae83b705a03714c46067a7aa57b02

Signed-off-by: Bernd Kuhls <bernd@kuhls.net>
---
 topology/topology.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/topology/topology.c b/topology/topology.c
index 1840ffe..5d03a8b 100644
--- a/topology/topology.c
+++ b/topology/topology.c
@@ -32,6 +32,7 @@
 #include <sys/stat.h>
 #include <getopt.h>
 #include <assert.h>
+#include <locale.h>
 
 #include <alsa/asoundlib.h>
 #include <alsa/topology.h>
-- 
2.39.2

